developer/src/kmc-ldml/src/compiler/keys.ts

```diff
@@ -130,7 +130,11 @@ export class KeysCompiler extends SectionCompiler {
 
       for (let lkflick of lkflicks.flick) {
         let flags = 0;
-        const to = sections.strs.allocAndUnescapeString(lkflick.to, true);
+        let cookedTo = lkflick.to;
+        // pull in string variables and markers
+        cookedTo = sections.vars.substituteStrings(cookedTo, sections);
+        cookedTo = sections.vars.substituteMarkerString(cookedTo);
+        const to = sections.strs.allocAndUnescapeString(cookedTo, true);
```

Comment:
This pattern is repeated, refactor? It is also similar in disp.ts but written nested for added confusions!